### PR TITLE
added new version of ES

### PR DESCRIPTION
**Pull request by vijay120**

### CODE REVIEW

File: source/userguide/getting_started.rst [hunk 1]
```diff
@@ -146,14 +146,14 @@ macOS:
...
-| Elasticsearch |  See instructions below to download Elasticsearch 6.7                                                  |
+| Elasticsearch |  See instructions below to download Elasticsearch 7.8                                                  |
 +---------------+--------------------------------------------------------------------------------------------------------+
 
 .. code-block:: shell
 
-   curl https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-6.7.0.tar.gz -o elasticsearch-6.7.0.tar.gz
-   tar -zxvf elasticsearch-6.7.0.tar.gz
-   ./elasticsearch-6.7.0/bin/elasticsearch
+   curl https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-7.8.0-darwin-x86_64.tar.gz -o elasticsearch-7.8.0.tar.gz
```
Comment (ritvikshrivastava) [inline source/userguide/getting_started.rst:154]: Should there be an option here for linux and Mac?

**Comment by ritvikshrivastava** [comment]
Minor nit: there's an extra semi-colon in line [177](https://github.com/cisco/mindmeld/pull/177/files#diff-8b6dac4e334d7ea19f2a3d7e4ae18922R177)

File: source/userguide/getting_started.rst [hunk 2]
```diff
@@ -146,35 +146,54 @@ macOS:
...
++---------------+-------------------------------------------------------------------------------+
+| Elasticsearch |  See instructions below to download Elasticsearch 7.8 natively or using docker|
++---------------+-------------------------------------------------------------------------------+
 
+Native onboarding:
 
-.. note::
+.. code-block:: shell
+
+   curl https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-7.8.0-linux-x86_64.tar.gz -o elasticsearch-7.8.0.tar.gz
```
Comment (vijay120) [inline source/userguide/getting_started.rst:186]:
```suggestion
   wget https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-7.8.0-linux-x86_64.tar.gz -o elasticsearch-7.8.0.tar.gz
```

**Comment by vijay120** [comment]
@minhtuev Verified by Alex the ubuntu commands work

**Pull request merged**